Translation/docker_compose/amd/gpu/rocm/set_env.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-License-Identifier: Apache-2.0
+
+export TRANSLATION_HOST_IP='192.165.1.21'
+export TRANSLATION_EXTERNAL_HOST_IP='direct-supercomputer1.powerml.co'
+export TRANSLATION_LLM_MODEL_ID="haoranxu/ALMA-13B"
+export TRANSLATION_TGI_LLM_ENDPOINT="http://${TRANSLATION_HOST_IP}:8008"
+export TRANSLATION_HUGGINGFACEHUB_API_TOKEN='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'
```

Comment:
Your personal API TOKEN should be hidden, and just leave as '${your_api_token}'.